templates/macros/assets.html

```diff
@@ -37,7 +24,7 @@
         <div class="asset-card__tags">
             {% if post.extra.bevy_versions %}
             <div class="asset-card__tag-list asset-card__bevy-versions">
-                <img src="/assets/bevy_icon_dark.svg" alt="Small bevy icon">
+                <img src="/assets/bevy_icon_dark.svg" title="Supported bevy versions">
```

Comment:
Nit: we need both the `alt` and `title`, as the `alt` is also rendered when the `img` can't be loaded.

(Sorry, I should have made that clearer in my comment)